dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -413,3 +419,35 @@ dashboard.header.updateTimestamp = function () {
     $('.project_updated_at').text("Not saved"); // TODO i18n
   }
 };
+
+function getSummarizedProgressForAnonymousUser () {
```

Comment:
Please document this method, including jsdoc comments describing the return value.